src/blog/2023/06/introducing-the-flowforge-community-forum.md

```diff
@@ -0,0 +1,23 @@
+---
+title: Introducing the FlowForge Community Forum
+subtitle: A Community Forum for support, inspiration, and knowledge sharing
+description: 
+date: 2023-06-09 12:00:00
```

Comment:
I think the target date for tomorrow should be better than today?

Reply:
Changing to days date.